main/settings.py

```diff
@@ -151,11 +151,22 @@
 # CORS
 CORS_ALLOWED_ORIGINS = get_list_of_str("CORS_ALLOWED_ORIGINS", [])
 CORS_ALLOWED_ORIGIN_REGEXES = get_list_of_str("CORS_ALLOWED_ORIGIN_REGEXES", [])
-CORS_ALLOW_CREDENTIALS = True
 
-CSRF_TRUSTED_ORIGINS = get_list_of_str("CSRF_TRUSTED_ORIGINS", [])
+CORS_ALLOW_CREDENTIALS = get_bool("CORS_ALLOW_CREDENTIALS", True)  # noqa: FBT003
+SECURE_CROSS_ORIGIN_OPENER_POLICY = get_string(
+    "SECURE_CROSS_ORIGIN_OPENER_POLICY",
+    "same-origin",
+)
+
+CSRF_COOKIE_SECURE = get_bool("CSRF_COOKIE_SECURE", False)  # noqa: FBT003
```

Comment:
I think it would be safer to have the default be `True` here, just in case we accidentally forget to set it on RC and prod.  Can ensure it's True locally in docker by adding `CSRF_COOKIE_SECURE: "False"` to the `x-environment` section of `docker-compose.yml`

Reply:
Can you also add this line to the `x-environment` section of docker-compose.yml?

```
  CSRF_COOKIE_SECURE: ${CSRF_COOKIE_SECURE:-False}
```